proto/internalapi/central/compliance_operator.proto

```diff
@@ -113,6 +113,18 @@ message ComplianceResponse {
     }
 }
 
+// Represents the severity of the rule
+enum ComplianceOperatorRuleSeverity {
+    UNSET_RULE_SEVERITY = 0;
+    UNKNOWN_RULE_SEVERITY = 1;
```

Comment:
Is unknown an explicit severity or can it be represented by 0 (unset)?

Reply:
Yes, unknown is an explicit coming from the operator.

Reply:
CO will have severity as unknown, so it can not be  represented by 0